### ISSUE TITLE
Support for spotify shorten url

### ISSUE THREAD

**Issue by maiconcarraro** [ctjlewis/react-spotify-embed#8]
Hey, first of all, great work!

I'd suggest to give support to the shorten urls example: https://spotify.link/TCO3AoyxkKb

**Comment by x5engine** [comment]
you could do a regexp for it too

**Comment by maiconcarraro** [comment]
@x5engine not possible, the only workaround is to do a `fetch` request and get the response url, which I'm doing in advance before it passes to the component, the suggestion is to have this out of the box.

**Comment by x5engine** [comment]
yeah, didnt know this wasnt supported by spotify, well then thats another fetch

**Comment by ctjlewis** [comment]
CORS blocks this.
